applications/StructuralMechanicsApplication/tests/cpp_tests/test_high_cycle_fatigue_constitutive_law_3d.cpp

```diff
@@ -0,0 +1,156 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                     license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Sergio Jiménez & Alejandro Cornejo
```

Comment:
Additionally can we have a more standard test?

Reply:
What do you mean by more standard test ? Tnx

Reply:
To a case that can be "generated" with the problemtype. With mdap and parameters

Reply:
OK, I will do it, but in a future PR if it is possible.